packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -164,19 +181,19 @@ export const Tooltip = ({
         ref={elementRef}
         className={targetWrapperClassName}
         onMouseEnter={(evt: MouseEvent) => {
-          setShow(true);
+          setIsHoveringTarget(true);
           if (onMouseOver) onMouseOver(evt);
         }}
         onMouseLeave={(evt: MouseEvent) => {
-          setShow(false);
+          setTimeout(() => setIsHoveringTarget(false), delay);
```

Comment:
@gui-santos I couldn't get it to work with no `setTimeout()`. The tooltip content gets hidden too quickly before I can move the cursor on to it. Do you see any drawbacks to using setTimeout() for this purpose?

Reply:
Nop! I forgot that it's impossible to move your mouse quickly enough to keep the tooltip open, sorry